core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -3251,21 +3307,24 @@ class WorkspaceServiceSpec
       ) should contain theSameElementsAs expected
   }
 
-  it should "not return a MC workspace that does not have a cloud context" in withTestDataServices { services =>
+  it should "not return MC workspaces that do not have a cloud context" in withTestDataServices { services =>
     val service = services.workspaceService
     val workspaceId1 = UUID.randomUUID().toString
     val workspaceId2 = UUID.randomUUID().toString
+    val workspaceId3 = UUID.randomUUID().toString
```

Comment:
Add in another Azure workspace without cloud context so we can catch both ways that the filtering happens (Ready workspace vs. non-Ready workspace).